app/views/presenters/_work_provenance.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= view.provenance_summary %>
+<% unless view.provenance_notes.nil? %>
+    <a data-toggle="collapse" href="#collapseProvenanceNotes"
+      role="button" aria-expanded="false"
+      aria-controls="collapseProvenanceNotes">Show notes</a>
+    <div class="collapse" id="collapseProvenanceNotes">
+      <%= view.provenance_notes %>
+    </div>
+<% end %>
+
+<%#
+  TODO: add css when we include this template.
+  #collapseProvenanceNotes {
+    margin-top: 1em;
+  }
```

Comment:
The CSS should be added in the same PR, no reason to leave a note explaining they need to be added instead of adding them. It's true that we don't have a CSS file for it yet; we can pick one now, can always change it later. Maybe `show_page.css` or `work_show.css` or `show_item.css`? I dunno, can always be changed, but we should get the CSS in now.